simapp/upgrades.go

```diff
@@ -25,6 +25,15 @@ func (app SimApp) RegisterUpgradeHandlers() {
 	app.UpgradeKeeper.SetUpgradeHandler(
 		UpgradeName,
 		func(ctx context.Context, _ upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
+			// sync accounts module and auth module account number
```

Comment:
@julienrbrt @tac0turtle where should the migration live?

it needs to set x/accounts Account number to auth's account number.

Reply:
if accounts does not have a dependency on auth but auth has a dep on accounts then I think it would be good for auth to have the migration and set it in the other module

Reply:
added the migration in